.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.761
+    hooks:
+      - id: mypy
+        args: [--show-error-codes, --ignore-missing-imports, --disallow-untyped-decorators, --disallow-incomplete-defs, --disallow-untyped-defs, --follow-imports, skip]
```

Comment:
Could you please keep the comments on which arguments we cannot use?

Reply:
Done!